shell/platform/fuchsia/flutter/accessibility_bridge_unittest.cc

```diff
@@ -127,6 +127,105 @@ TEST_F(AccessibilityBridgeTest, DeletesChildrenTransitively) {
   EXPECT_FALSE(semantics_manager_.UpdateOverflowed());
 }
 
+TEST_F(AccessibilityBridgeTest, PopulatesCheckedState) {
+  flutter::SemanticsNode node0;
+  node0.id = 0;
+  // HasCheckedState = true
+  // IsChecked = true
+  // IsSelected = false
+  // IsHidden = false
+  node0.flags = static_cast<int>(flutter::SemanticsFlags::kHasCheckedState) + static_cast<int>(flutter::SemanticsFlags::kIsChecked);
```

Comment:
Shouldn't this be `|` instead of `+`?

This may also be a bit more readable as 

```
node.flags |= static_cast<int>(flutter::SEmanticsFlags::kHasCheckedState);
node.flags |= ...
```